### ISSUE TITLE
[Bug]: Unable to install from homebrew tap, the download URL is incorrect

### ISSUE THREAD

**Issue by dekimsey** [russellbanks/Komac#425]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

When attempting to install the currently available release via hombrew (2.0.0), the install fails. While I noticed that `2.0.0` had a minor mishap with the artifact names (published as nightly) I wanted to highlight the fact that the corrected release assets use a tag and not the version in their names, so the correct asset URL would have been `KomacPortable-macos-v2.0.0-...` (note the `v` prefix on the version). Just highlighting this so when the tap gets updated, the URL will be correct.

```sh
 brew install russellbanks/tap/komac
==> Fetching russellbanks/tap/komac
==> Downloading https://github.com/russellbanks/Komac/releases/download/v2.0.0/KomacPortable-macos-2.0.0-arm64.tar.gz
curl: (22) The requested URL returned error: 404

Error: komac: Failed to download resource "komac"
Download failed: https://github.com/russellbanks/Komac/releases/download/v2.0.0/KomacPortable-macos-2.0.0-arm64.tar.gz

```

~~Also, v2.0.2 is released and the tap is a couple versions out of date <3.~~ edit: these are nightlies according to winget

Unfortunately, I don't have any prior art to recommend for adding a GH action to the release pipeline to help keep the tap up to date. From my experience, there are two routes:
  - Having this repo's release workflow create a PR in the tap repo and then merging it.
  - Using repo_dispatch/workflow_dispatch to trigger a workflow in the tap and providing the version as an input. The tap repo workflow would do the rest. That does mean it's possible for one to control or independently re-run the tap workflow which is nice. But this route means one has to jump through a bit more hoops to track completion of the job since GH's API doesn't tell you the job ID.

Thank you!

**Comment by vedantmgoyal2009** [comment]
As https://github.com/russellbanks/Komac/pull/418 was merged after v2.0.2, I'm waiting for v2.0.3. Once it gets released, I'll have a CI setup for it.

**Comment by vedantmgoyal2009** [comment]
They are working now.
